### PR TITLE
Popups in IE need to load a document

### PR DESCRIPTION
**Pull request by mightyiam**
If `window.open` in IE does not load a document,
the parent window may not be allowed to control the child,
due to some "same-origin domain policy" security issue.

This results in entirely unusable popup windows.

This may be the cause of the following from the customer support room:
screenshot_2015-08-15_01-53-35

### CODE REVIEW

File: src/wymeditor/popup.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<!-- Workaround for IE https://github.com/wymeditor/wymeditor/issues/731 -->
```
Comment (mightyiam) [inline src/wymeditor/popup.html]: So we make a blank document.

File: src/wymeditor/editor/dialogs.js [hunk 2]
```diff
@@ -86,11 +107,7 @@ WYMeditor.editor.prototype.dialog = function (
         ].join(",");
     }
 
-    wDialog = window.open(
-        '',
-        "wymDialogWindow",
-        dialogWindowFeatures
-    );
+    wDialog = wym._openPopupWindow(dialogWindowFeatures)
```
Comment (mightyiam) [inline src/wymeditor/editor/dialogs.js]: We make this into a method.

File: src/wymeditor/editor/dialogs.js [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+    Opens a popup window, provided the `windowFeatures`.
+*/
+WYMeditor.editor.prototype._openPopupWindow = function (windowFeatures) {
+    var wym = this,
+        popup,
+        basePath = wym._options.basePath;
+
+    popup = window.open(
+        // https://github.com/wymeditor/wymeditor/issues/731
+        jQuery.browser.msie ? basePath + 'popup.html' : '',
```
Comment (mightyiam) [inline src/wymeditor/editor/dialogs.js]: All other browsers do no have this issue thus are fine with a blank string here.

File: src/wymeditor/core.js [hunk 4]
```diff
@@ -936,7 +936,7 @@ WYMeditor._computeWymPath = function () {
     the wymeditor base file. This path is used as the basis for loading:
     * Language files
     * Skins
-    *
+    * The popup document https://github.com/wymeditor/wymeditor/issues/731
```
Comment (mightyiam) [inline src/wymeditor/core.js]: There was an asterisk here, waiting for me.

File: Gruntfile.js [hunk 5]
```diff
@@ -360,6 +360,13 @@ module.exports = function (grunt) {
                             "iframe/{,*/}*.{html,js,css,png,jpg,jpeg,gif,eot,ttf,woff}"
                         ]
                     },
+                    // Popup document
+                    // https://github.com/wymeditor/wymeditor/issues/731
+                    {
+                        cwd: '<%= yeoman.app %>/wymeditor',
+                        dest: '<%= yeoman.dist %>/wymeditor',
+                        src: "popup.html"
+                    },
```
Comment (mightyiam) [inline Gruntfile.js]: This includes the popup document in the distribution.

Comment (mightyiam) [inline Gruntfile.js]: I tested that the distribution works, as well.

**Comment by jlward** [comment]
Looks good.

**Comment by mightyiam** [comment]
Thank you, @jlward.

**Pull request merged**